islandora-indexing-fcrepo/src/main/java/ca/islandora/alpaca/indexing/fcrepo/FcrepoIndexer.java

```diff
@@ -197,11 +212,16 @@ public void configure() {
                 .setProperty("uuid").simple("${exchangeProperty.event.object.id.replaceAll(\"urn:uuid:\",\"\")}")
                 .setProperty("drupal").simple("${exchangeProperty.event.object.url[0].href}")
                 .setProperty("fedora").simple("${exchangeProperty.event.attachment.content.fedoraUri}")
```

Comment:
Pretty sure you can axe this and use 'target' instead.

Reply:
No, you need both...  "fedoraUrl" is the base url for fedora, and "fedora" is the full url to the resource.